src/App.test.js

```diff
@@ -1,8 +1,22 @@
 import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom'
 import App from './App';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+describe('Check that all components presented on a page', () => {
+  test('Check counter exist', () => {
+    render(<App />);
+    const linkElement = screen.getByText(/Count/i);
```

Comment:
Note: Sometimes, destruction queries from `render` is more concise and convenient.